doc/classes/Projection.xml

```diff
@@ -40,12 +47,14 @@
 			<return type="Projection" />
 			<param index="0" name="flip_y" type="bool" />
 			<description>
+				Creates a new [Projection] that projects positions with an adjusted depth of field, and flips the projected positions vertically, according to [param flip_y].
 			</description>
 		</method>
 		<method name="create_fit_aabb" qualifiers="static">
 			<return type="Projection" />
 			<param index="0" name="aabb" type="AABB" />
 			<description>
+				Creates a new [Projection] that projects positions into the given [AABB].
```

Comment:
```suggestion
				Creates a new [Projection] that scales a given projection to fit around a given [AABB] in projection space.
```

This is an odd one. It seems to be a leftover from Godot 2.x (it hasn't been used since then). It was only used for a single purpose, so it may not even be useful anymore.

Reply:
What do you mean when you say "scales a given projection"? Is it mean to be multiplied with another `Projection`, and not used on its own?

Reply:
Yes, the only place I can find it being used is to scale another matrix that will be used later in the rendering pipeline. I'm not sure it would be useful to multiply the resulting matrix by anything else.